notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1549 @@
+{
```

Comment:
<p>"We define a simple neural network model, and training and validation steps in Pytorch." -&gt; "We will define a simple neural network model with the training and validation steps in PyTorch"</p><p> </p><p>"In Pytorch, we can define a neural network model in a class subclassing <code>torch.nn.Module</code>.</p><p> &nbsp;there is an <code>__init__()</code> method that defines the layers and other components of a model, and a <code>forward()</code> method where the computation gets done." -&gt; "In Pytorch, we can define a neural network model using class subclassing <code>torch.nn.Module</code>.</p><p> &nbsp;There is an <code>__init__()</code> method that defines the layers and other components of a model, and a <code>forward()</code> method where the computation gets done."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I modified the second sentence in different way since "using class subclassing&nbsp;<code>torch.nn.Module</code>" sounds a bit strange to me. </p>